### PR TITLE
Add extra spaces for connector configuration field examples

### PR DESCRIPTION
**Pull request by tuliren**
## Summary
- Currently the example in the connection configuration is always surrounded by a pair of parentheses.
  example
    - The screenshot is from a user connector setup page.
- When user copies the example, they can easily include the ending parenthesis if not careful, and the ending parenthesis can lead to configuration issue. See https://github.com/airbytehq/oncall/issues/465 for example.
- This PR adds extra spaces in the locale file to solve this problem.
- Relate to https://github.com/airbytehq/oncall/issues/465.

### CODE REVIEW

File: airbyte-webapp/src/locales/en.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
   "form.dockerError": "Could not find docker image",
   "form.edit": "Edit",
   "form.done": "Done",
-  "form.examples": "(e.g. {examples})",
+  "form.examples": "( e.g. {examples} )",
```
Comment (lmossman) [inline airbyte-webapp/src/locales/en.json:94]: Would it potentially be better to just remove the parentheses completely, if that avoids bad copy-pastes entirely?

Comment (tuliren) [reply airbyte-webapp/src/locales/en.json:94]: That's possible. What about
```suggestion
  "form.examples": "E.g. {examples}"
```

The minor issue is that there is no closing full stop at the end. So technically it is wrong if we ask a high school grammar teacher. But there is no full stop currently already. So I think that can be ignored.

Comment (lmossman) [reply airbyte-webapp/src/locales/en.json:94]: Yeah I think this should be fine!

**Review by lmossman — APPROVED** [review]

**Comment by lmossman** [comment]
@tuliren I am actually working on redesigning these field messages completely (putting them in a tooltip, etc), so I am going to close out this PR in favor of that change. I will be sure to format the examples nicely so that they do not have parentheses like the original problem here

**Comment by tuliren** [comment]
Cool. Thank you for letting me know. Looking forward to the new design.

**Pull request closed without merging**